api/client.go

```diff
@@ -30,20 +32,17 @@ type (
 		BaseURL string
 
 		// Services used for talking with different parts of the Recurly API
-		Accounts      *AccountsService
-		Adjustments   *AdjustmentsService
-		Billing       *BillingService
-		Coupons       *CouponsService
-		Redemptions   *RedemptionsService
-		Invoices      *InvoicesService
-		Plans         *PlansService
-		AddOns        *AddOnsService
-		Subscriptions *SubscriptionsService
-		Transactions  *TransactionsService
+		Accounts      recurly.AccountsService
```

Comment:
Fixed in [b74efc](https://github.com/blacklightcms/go-recurly/pull/10/commits/9bea2c2253c260f6d17ac6dd436edd7566b74efc)